src/main/java/io/codearte/jfairy/Bootstrap.java

```diff
@@ -94,13 +105,40 @@ public static Fairy create(Locale locale, String dataFilePrefix) {
 				.build();
 	}
 
+
+	public static Fairy create(Provider<DataMaster> dataMaster, Locale locale) {
+		return builder().withDataMasterProvider(dataMaster).withLocale(locale).build();
+	}
+
+	private static FairyModule getFairyModuleForLocale(DataMaster dataMaster, Locale locale, Random random) {
+		LanguageCode code = LanguageCode.valueOf(locale.getLanguage().toUpperCase());
+		switch (code) {
+			case PL:
+				return new PlFairyModule(dataMaster, random);
+			case EN:
+				return new EnFairyModule(dataMaster, random);
+			case ES:
+				return new EsFairyModule(dataMaster, random);
+			default:
+				return new EnFairyModule(dataMaster, random);
```

Comment:
This is also -and most predominantly- being used when no locale has been provided at all. So I would log "Using defualt locale: EN". What do you think?